src/tools/wasm-split/CMakeLists.txt

```diff
@@ -6,5 +6,5 @@ set(wasm_split_SOURCES
 )
 add_library(wasm-split-lib OBJECT ${wasm_split_SOURCES})
 
-binaryen_add_executable(wasm-split wasm-split.cpp)
+binaryen_add_executable(wasm-split wasm-split.cpp $<TARGET_OBJECTS:wasm-split-lib>)
 target_link_libraries(wasm-split wasm-split-lib)
```

Comment:
I don't think it makes sense to add the object files from `wasm-split-lib` as wells the library itself.

Surely `target_link_libraries` entire purpose is to include the library so I don't see why this change would be needed?

Reply:
Oh right, I forgot to remove the `target_link_libraries`.

Reply:
Yeah, my interpretation of the [doc](https://cmake.org/cmake/help/latest/command/add_library.html) is that if we use this TARGET_OBJECTS thing, it would be instead of target_link_libraries rather than in addition to it.